ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -4,7 +4,8 @@
 from mlagents.trainers.buffer import Buffer
 from mlagents.envs.brain import BrainParameters, BrainInfo
 from mlagents.envs.communicator_objects import *
-from google.protobuf.internal.decoder import _DecodeVarint32
+from google.protobuf.internal.decoder import _DecodeVarint32  # type: ignore
```

Comment:
What does #type:ignore mean here?

Reply:
`_DecodeVarint32` isn't declared in the protobuf library stubs here: https://github.com/python/typeshed/blob/master/third_party/2and3/google/protobuf/internal/decoder.pyi
I'm not sure if this is an oversight, or because they don't want to provide type info for internal functions (it starts with a `_`), or because it's not available in all versions of protobuf. 

`# type: ignore` just tells mypy to ignore errors on that line; it will treat `_DecodeVarint32` as an `Any`. We can try to fix this up, but it didn't seem worth the effort.

Reply:
I agree, this is good.